multi_asset/multi_token_impl.mligo

```diff
@@ -29,7 +29,14 @@ let add_operator (operator : address) (approvals : approvals) : approvals =
   let new_operators =
     match Map.find_opt sender approvals with
     | Some(ops) -> Set.add operator ops
-    | None      -> Set.literal [operator]
+    | None      ->
+        (* 
+          Check that sender implements `multi_token_receiver` interface.
+          If not, `get_contract` will fail
+        *)
+        let receiver : multi_token_receiver contract = 
+          Operation.get_contract sender in
+        Set.literal [operator]
```

Comment:
This seems unhelpful, and unnecessary.

Reply:
It is preventing a contract which does not implement `multi_token_receiver` from registering its operators. Strictly, it is not necessary since `transfer` function will do the check anyway, but it is an extra guard